test/expectations/hover/documented_namespaced_class.exp.json

```diff
@@ -17,7 +17,7 @@
             },
             "end": {
                 "line": 1,
-                "character": 14
```

Comment:
Because of the known off-by-one issue in YARP which will be addressed separately.